tests/test_rule_handling.py

```diff
@@ -657,37 +656,52 @@ def test_evaluation_various_rules(proxy_getter, rule_data, ksdata_mock, storage_
 
 def test_revert_mount_options_nonexistent(proxy_getter, rule_data, ksdata_mock, storage_mock):
     rule_data.new_rule("part /tmp --mountoptions=nodev")
-    storage_mock.mountpoints = dict()
 
     messages = rule_data.eval_rules(ksdata_mock, storage_mock)
 
     # mount point doesn't exist -> one message, nothing done
     assert len(messages) == 1
-    assert storage_mock.mountpoints == dict()
+
+    device_tree_mock = STORAGE.get_proxy(DEVICE_TREE)
+    device_tree_mock.SetDeviceMountOptions.assert_not_called()
```

Comment:
Does this assert tell us anything? The device tree has just been created, so testing that a method of itself was not called doesn't look like something to be tested in the addon code.

Reply:
The `proxy_getter` fixture makes sure that `STORAGE.get_proxy(DEVICE_TREE)` always returns the same mock object. Therefore, this mock object might be already used in the `eval_rules` method. I can move the definition of the `device_tree_mock` variable at the beginning of the test if it is too confusing.